gdeployfeatures/vg/vg.py

```diff
@@ -46,9 +46,7 @@ def set_default_values(section_dict, pvnames, vgnames):
                 print "Error: %s"%msg
                 Global.logger.error(msg)
                 helpers.cleanup_and_quit()
-            if section_dict['one-to-one'] == 'no':
-                vgnames *= len(pvnames)
-            else:
+            if section_dict['one-to-one'] == 'yes':
```

Comment:
@devyanikota you can remove this one-to-one option altogether. 
Because now we provide provision to add more than one pv to a vg by default.

So it is one-to-one if vgnames == pvnames (which we do anyway)
Else if vgnames == 1 and < pvnames we aggregrate.

So, one-to-one does not carry use except checking an extra condition. Can you review and remove it?

Reply:
@sac initially I thought we would be losing the feature of providing just one vgname and getting two vgs with their respective pvs, but there is no such feature in the manual creation either.
Now that I rethink, I think its best to remove this condition in order to avoid confusion, keeping in sync with the manual options.
Thank you.